script.js

```diff
@@ -1,4 +1,238 @@
+// Random Dice Roll Generator
+var randomDiceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomDiceNum = randomInteger + 1;
+  return randomDiceNum;
+};
+
+// Global Variables for Normal Dice Game
```

Comment:
Global vars typically declared at top of file by convention, above helper functions